python/lsst/meas/algorithms/stamps.py

```diff
@@ -370,8 +378,15 @@ def writeFits(self, filename):
         """
         self._refresh_metadata()
         type_name = get_full_type_name(self)
-        writeFits(filename, self._stamps, self._metadata, type_name, self.use_mask, self.use_variance,
-                  self.use_archive)
+        writeFits(
+            filename,
+            self._stamps,
+            self._metadata,
+            type_name,
+            self.use_mask,
+            self.use_variance,
+            self.use_archive,
+        )
```

Comment:
I don't quite see why to call `writeFits` method that was defined earlier in the file, rather than implementing the method here. If you agree with here, create a ticket (perhaps the same one that refactors `readFitsWithOptions`) and comment with TODO.

Reply:
The `writeFits` function at the top of the file is a free function, whereas this `writeFits` method is a shortcut for writing a `StampsBase` class type. I guess the original design intention was for to allow this free function to be called and used with generic stamp objects?

Reply:
Following our in-person discussion, I think we're happy to leave this as-is for now.